wiki/Chat_Console/id.md

```diff
@@ -89,7 +89,7 @@ Tekan tombol angka atau menekan salah satu bar untuk:-
+| `/watch [pengguna]`           | Memulai menonton `[pengguna]`.                                                                                                                                                                                                     | `/watch Amigo`                     | Started spectating Amigo. (Jika Amigo bermain sebuah beatmap yang kamu miliki, kamu akan menonton permainan dia [setelah beberapa saat pemuatan] dengan namamu di sebelah kiri layar Amigo)                                                                                                                                                                                                                                                                              |
 | `/nopm`                       | Memperbolehkan atau tidaknya pesan pribadi dari semua orang                                                                                                                                                                      | `/nopm`                            | (Sebuah akan muncul di tengah yang menjelaskan bahwa kamu memperbolehkan/tidaknya pesan pribadi dari semua orang)                                                                                                                                                                                                                                                                                                                                                       |
 | `/invite [pengguna]`          | `[pengguna]` telah di undang ke dalam permainan                                                                                                                                                                                          | `/invite Nathanael`                | * Nathanael has been invited to the game                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
> | `/invite [pengguna]`          | `[pengguna]` telah di undang ke dalam permainan                                                                                                                                                                                          | `/invite Nathanael`                | * Nathanael has been invited to the game                                                                                                                                                                                                                                                                                                                                                                                                                                |

use `diundang` because `di` in this context doesn't to point a place but a verb